app/tests/Date_Tests.cpp

```diff
@@ -1,90 +0,0 @@
-#include <catch2/catch.hpp>
-#include <string.h>
-
-#include "../src/Date.hpp"
-
-
-TEST_CASE("Normal Date check", "[Date]"){
-    
-    try{
-        Date* date = new Date(2021,1,14);
-        REQUIRE(true);
```

Comment:
i just really dont like this date setting choice. "2021-1-14" or "2021-01-14" will be wayyy easier to work with down the line. and then you can just have a "getYearMonthDayFromDate" method to get your numbers if you need them.